GPU/Common/FramebufferManagerCommon.h

```diff
@@ -512,15 +512,15 @@ class FramebufferManagerCommon {
 	void EstimateDrawingSize(u32 fb_address, int fb_stride, GEBufferFormat fb_format, int viewport_width, int viewport_height, int region_width, int region_height, int scissor_width, int scissor_height, int &drawing_width, int &drawing_height);
 
 	void NotifyRenderFramebufferCreated(VirtualFramebuffer *vfb);
-	void NotifyRenderFramebufferUpdated(VirtualFramebuffer *vfb);
+	static void NotifyRenderFramebufferUpdated(VirtualFramebuffer *vfb);
 	void NotifyRenderFramebufferSwitched(VirtualFramebuffer *prevVfb, VirtualFramebuffer *vfb, bool isClearingDepth);
 
 	void BlitFramebufferDepth(VirtualFramebuffer *src, VirtualFramebuffer *dst);
 
 	void ResizeFramebufFBO(VirtualFramebuffer *vfb, int w, int h, bool force = false, bool skipCopy = false);
 
-	bool ShouldDownloadFramebufferColor(const VirtualFramebuffer *vfb) const;
-	bool ShouldDownloadFramebufferDepth(const VirtualFramebuffer *vfb) const;
+	static bool ShouldDownloadFramebufferColor(const VirtualFramebuffer *vfb) ;
```

Comment:
Did you use clang-format?
`(const VirtualFramebuffer *vfb) ;`
`-------------------------------^`

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18986/commits/b57dab28126ec2ee5d9d62d9e6e37b5b756e667f